indico/modules/rb/client/js/util.jsx

```diff
@@ -161,19 +161,26 @@ export function renderRecurrence({type, number, interval}) {
   }
   if (type === 'single') {
     return (
-      <Responsive.Tablet andLarger orElse="S">
+      <Responsive.Tablet andLarger orElse={<Translate context="single">S</Translate>}>
```

Comment:
i think the context should be something more verbose like "single booking shortcut".

fyi, `Translate.string()` also has a context arg, so you don't need to use the jsx version here